src/libstd/io/error.rs

```diff
@@ -95,6 +95,9 @@ pub enum ErrorKind {
     /// A parameter was incorrect.
     #[stable(feature = "rust1", since = "1.0.0")]
     InvalidInput,
+    /// Invalid data encountered.
+    #[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
This stability tag should now say `1.1.0`, and the feature name can be something like `io_invalid_data`.

Could you also expand the docs here a bit to clarify what `InvalidData` means with respect to `InvalidInput`?